tpe/plugin/src/SimulationFeatures.cc

```diff
@@ -98,6 +98,40 @@ void SimulationFeatures::Write(ChangedWorldPoses &_changedPoses) const
     }
   }
 
+  // Iterate over models to make sure link velocities for moving models
+  // are calculated and sent
+  for (const auto &[id, info] : this->models)
+  {
+    // make sure the model exists
+    if (info)
+    {
+      if (info->model->GetStatic())
+        continue;
+      const auto nextPose = info->model->GetPose();
+      // Note, now prevLinkPoses also containes prevModelPoses
+      // Data structure has been kept to avoid breaking ABI
```

Comment:
`containes` -> `contains`

About ABI - I think we can change the data structure without breaking ABI since `SimulationFeatures.h` is in the `tpe/plugin/src` directory and it's not installed.

So maybe also rename `prevLinkPoses` to `prevEntityPoses`?